fixtures/flight/package.json

```diff
@@ -70,7 +70,7 @@
     "predev": "cp -r ../../build/oss-experimental/* ./node_modules/",
     "prebuild": "cp -r ../../build/oss-experimental/* ./node_modules/",
     "dev": "concurrently \"npm run dev:region\" \"npm run dev:global\"",
-    "dev:global": "NODE_ENV=development BUILD_PATH=dist node --experimental-loader ./loader/global.js server/global",
+    "dev:global": "NODE_ENV=development BUILD_PATH=dist node --experimental-loader ./loader/global.js --inspect server/global",
```

Comment:
Just a small heads-up: we might want to select a specific port here to avoid conflicts with the regional server when this and #30140 are both merged.

Reply:
Doesn't Node.js automatically select a free port for the debugger?

Reply:
No, if not otherwise specified it will try to use `9229`, and fail with `Starting inspector on 127.0.0.1:9229 failed: address already in use` if it's already in use.